cmd/kubectl-package/build/build.go

```diff
@@ -0,0 +1,99 @@
+package build
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	internalcmd "package-operator.run/package-operator/internal/cmd"
```

Comment:
The reason for the original naming is to avoid named imports. k8s is full of this and my favorite example is the following:

```
	schemacell "k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel"
	pkgapiscell "k8s.io/apiserver/pkg/apis/cel"
	pkgcell "k8s.io/apiserver/pkg/cel"
```

gopls and a lot of IDEs have trouble to autoresolve dependencies when every package is basically named the same so I am always trying to give it a less ambiguous name.